### DISCUSSION TITLE
Creating a Fresh Install  with clean Database

### DISCUSSION THREAD

**Discussion by topgunisp** [stustapay/stustapay#263] [Q&A]
hello I have managed to get the service up and running using some of the test DB 

I have the following questions ??

how can I set up administrative access to avoid cluttering the database with test information."

"Why are these fields needed? What data should be there?
secret_key: "very secret"
sumup_affiliate_key: "pretty secret"

I want to create a clean install with no test data 

thank you for all your help

**Comment by TheJJ** [comment]
hi! as long as you don't feed in the test data, you won't clutter your db :)
the secret key is the one used for the mifare ultralight aes with the required programming, the sumup affiliate key you need when using sumup for EC card payments to top up the balance of a wristband/card.

to fill the db with its structures without test data, just apply the db migrations :)

**Reply by topgunisp** [reply]
Thank you for the quick reply 

could you please tell me what type of programming it’s required for the Mifare Ultralight aes ?? 

referring to the NFC wristband before I could use them with the system do I need to registered them ??  on the system ?

All your answers are very helpful

**Reply by mhcp6** [reply]
Yes, you need to register them in the System in the DB table 'user_tag' with their tag UID and the assigned PIN.
Tags that are not in this list are refused.

For testing you can fill in this data "by hand".
For production you should have a list from the tag manufacturer with this data you feed in the DB once.

'serial' is a custom tag id that was programmed, but not used.
'secret' refers to the secret mentioned above in table 'user_tag_secret'

For programming the tags, I have to refer to somebody else.
But the app should have a debug mode, where you can program tags yourself (=set the secret and lock them).
